src/routes/auth/logout.route.js

```diff
@@ -0,0 +1,22 @@
+const User = require('../../entities/user')
+
+module.exports.post = {
+	method: 'POST',
+	path: '/logout',
+	schema: {},
+	handler: async (req, res, _next) => {
+		const { refreshToken } = req.body
```

Comment:
soon, all requests coming to the api that are **authenticated** will require a access token in the header of each request, in the form of:

`Authorization`: `bearer eyJhb...`

so we basically are going to be able to take that token, verify its signature (to make sure its not counterfeit), verify that it's still valid and not expired, and then get the user bound to the token from the db. from there passport "attaches" to the `req` object a lot of stuff, like a `user` field, where we can access all user's info, and a `logout` method, which you can just call from here like 
```
req.logout(function(err) {
    if (err) return next(err)
    res.status(200).send('Logged out')
}
```